.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -29,6 +29,36 @@ assignees: freitagb, wildintellect
 **Intended science use case**
 *Please describe how you intend to use the data, or the expected relevance to MAAP users.*
 
+## Spatio-Temporal Asset Catalog (STAC)
+
+This collection will be published as a Spatio-Temporal Asset Catalog (STAC) Collection. You can read the complete STAC collection spec here: (https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md).
+
+If you need help with any of the fields below, please let us know.
```

Comment:
Please fill in any fields you know. If you don't know the answer it can be updated later in consultation with the Data Team.